### DISCUSSION TITLE
Version 3.0.1

### DISCUSSION THREAD

**Discussion by nimrod-gileadi** [google-deepmind/mujoco#1215] [Announcements]
On November 15, we pushed a follow-up to the [MuJoCo 3.0.0](https://github.com/google-deepmind/mujoco/discussions/1101) release. [Version 3.0.1](https://github.com/google-deepmind/mujoco/releases/tag/3.0.1) has a few bug fixes, and some features that didn't quite make it in time into 3.0.

Take a look at the [change log](https://mujoco.readthedocs.io/en/3.0.1/changelog.html#version-3-0-1-november-15-2023) and let us know what you think!

**Comment by tmplt** [comment]
Thanks for the added colab tutorials. They are much appreciated.
